.github/workflows/manage_title.yml

```diff
@@ -0,0 +1,26 @@
+name: Bounty hunter issue label assignment management
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  extract_filename_from_path:
+    name: "Generate the title from the filename"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the spec filename
+        uses: actions-ecosystem/action-regex-match@v2
+        id: match-spec-filename
+        with:
+          text: ${{ github.event.issue.body }}
+          regex: '^\s*https:\/\/github\.com\/.*\/bounties\/level-([\d])+\/(.*)\.md$'
+          flags: m
+      - name: Add label for bounty level
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          labels: |
```

Comment:
This will add three labels to the issue `bounty`, level (e.g. `level 1`), and bounty spec name (e.g. `create-bounties`).